share/completions/emaint.fish

```diff
@@ -18,11 +18,11 @@ complete -c emaint -n '__fish_use_subcommand' -xa 'sync'        -d 'Check repos.
 complete -c emaint -n '__fish_use_subcommand' -xa 'world'       -d 'Check and fix problems in the world file'
 
 ## Local opts
-# cleanlogs
-complete -c emaint -n '__fish_seen_subcommand_from cleanlogs' -s t -l time      -d "Delete logs older than NUM of days" \
+# logs
+complete -c emaint -n '__fish_seen_subcommand_from logs' -s t -l time      -d "Delete logs older than NUM of days" \
```

Comment:
That `of` feels weird to me - `Delete logs older than NUM days`.

Reply:
`of`?

Reply:
In the description: "Delete logs older than NUM **of** days"

Reply:
I swear I read that line like eight times and did not see an `of` in there. You are right though, the wording is weird.